Packs/SplunkPy/Scripts/SplunkPySearch/SplunkPySearch.py

```diff
@@ -0,0 +1,21 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+query = demisto.args()['query']
```

Comment:
Just to make sure, no changes to the code here right?

Reply:
Hey @YuvHayun , No changes here